### PR TITLE
feat: updated styles for button atom component

### PR DESCRIPTION
**Pull request by TeddyAndersson**
## Feature description
Updates the Button atom component to follow the user interface design. This includes the option to render Contained buttons, Outlined Buttons and setting correct sizing 

Example of component usage:
```JSX 
<Button
      colorSchema="purple" // default value is blue. 
      variant="outlined" // default value is contained
      onClick={(event) => ...do something}
      size="small" // default value is medium
      disabled={false}
      z={1} // elevation of the button
    />
```
More examples can be found in our Storybook.

## How to test the feature?

1. Checkout branch
2. Swap the application to run storybook
3. Fire upp the simulator by running the command`yarn ios`
4. Look in the story named Button
5. Compare to design in Figma or look at attachement

Screen Shot 2020-11-05 at 11 48 39

## Is there any existing behaviour change of other features due to this code change?
No.

## Covered unit tests cases / E2E test cases?
No.

## Are your code structured in a way so that reviewers can understand it?
Yes.

## Was this feature tested in the following environments?
- [x] Storybook on a iOS device/simulator.

### CODE REVIEW

**Comment by zilaei** [comment]
Is the story (and style for) Buttons -> Pill obsolete?

**Review by jwinding — APPROVED** [review]
Looks great, approved!

(The only comment, which I don't feel like we should spend time on now so maybe it's stupid to even mention, is that size numbers perhaps should live in the theme? But there's better things to spend time on right now, so..)

**Comment by TeddyAndersson** [comment]
> @TeddyAndersson: Is the story (and style for) Buttons -> Pill obsolete?

We do not use pill anywhere, and if you follow the Figma design there is no mention of pills :)
If there is a need for a pill button my suggestion is that we add that as a variant type

**Comment by TeddyAndersson** [comment]
> Looks great, approved!
> 
> (The only comment, which I don't feel like we should spend time on now so maybe it's stupid to even mention, is that size numbers perhaps should live in the theme? But there's better things to spend time on right now, so..)

Good note @jwinding will add this in the backlog for future improvement as an overall task ie check what could be moved to the theme from components :)

**Comment by TeddyAndersson** [comment]
@zilaei if you approve of these changes i will rebase them.

**Review by zilaei — APPROVED** [review]

**Comment by zilaei** [comment]
> > @TeddyAndersson: Is the story (and style for) Buttons -> Pill obsolete?
> 
> We do not use pill anywhere, and if you follow the Figma design there is no mention of pills :)
> If there is a need for a pill button my suggestion is that we add that as a variant type

Thought about removing the story as it's not working now (as far as I can see). Like you say, we have no design or need for a pill button. 

This is a minor comment, do not spend to much time on this. :)

File: source/components/atoms/Button/Button.js [hunk 1]
```diff
@@ -4,70 +4,82 @@ import styled, { css } from 'styled-components/native';
...
+/* Styles for different button variants outlined, contained etc */
+Styles.outlined = css`
+  border: 2px solid ${props => props.theme.colors.primary[props.colorSchema][1]};
+  background-color: ${props => props.theme.colors.complementary[props.colorSchema][1]};
+  ${Styles.elevation[0]}
 `;
 
-CSS.buttonSharp = css`
-  border-radius: 0px;
+Styles.contained = css``;
```
Comment (dannil76) [inline source/components/atoms/Button/Button.js:28]: What does this do?

**Review by dannil76 — APPROVED** [review]

**Pull request merged**